src/components/settings-menu/PrivacySection.tsx

```diff
@@ -8,6 +7,13 @@ import useExperimentalFlag, { PROFILES } from '@/config/experimentalHooks';
 import { useAccountProfile, useShowcaseTokens, useWebData } from '@/hooks';
 import { useNavigation } from '@/navigation';
 import Routes from '@/navigation/routesNames';
+import Restart from 'react-native-restart';
```

Comment:
Are you restarting here because the logger doesn't have a `disable()` method? Let's just add disable/enable like analytics. That way we can avoid yet another dependency, and gives the user a better experience.

Can be in this PR, and add tests and all that. Should be super quick, lmk if I can help with Jest stuff 👍